hibernate-core/src/test/java/org/hibernate/orm/test/batchfetch/DynamicBatchFetchTest.java

```diff
@@ -33,6 +35,7 @@
 @ServiceRegistry(
 		settings = @Setting(name = AvailableSettings.USE_SECOND_LEVEL_CACHE, value = "false")
 )
+@SkipForDialect(dialectClass = AltibaseDialect.class, reason = "Altibase needs more memory setting to run this test")
```

Comment:
Can't you run your tests with more memory to avoid the skip?

Reply:
I thought tests should be executed in default database settings. So I'm gonna change database setting and if tests runs ok will remove this skip.